package/src/wallets/keplr/KeplrWallet.ts

```diff
@@ -118,12 +132,91 @@ export class KeplrWallet implements Wallet {
             sourceChannel: ibcChannels[tx.src][tx.dst],
             sender: this.address,
             receiver: tx.address,
-            token: tx.coin,
+            token: {
+              amount: tx.coin.amount.toFixed(0),
```

Comment:
chain wants amount to be string, must be checked for all messages in the sdk.
TODO @alecande11